tests/functional/z_cluster/cluster_expansion/test_resize_osd.py

```diff
@@ -78,6 +115,32 @@ def setup(self, create_pvcs_and_pods):
             pvc_size=pvc_size, num_of_rbd_pvc=5, num_of_cephfs_pvc=5
         )
 
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Check that the new osd size has increased and increase the resize osd count
```

Comment:
Logically, this validation should be considered a test step and not teardown

Reply:
This validation should also be performed if the test fails in one of the steps so we have the information for the next test. Maybe I can move it to a function in the `osd_resize` file.

Reply:
Agreed. Shouldn't we call it from the test as well?

Reply:
I don't think so, because then we call it twice. This check is more important for the following test cases.
If you are worried about the storage size check, it's already implemented here: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/osd_resize.py#L154.